logging/log_config.go

```diff
@@ -25,7 +25,8 @@ const (
 	validResourceSubTypePattern = validNamespacePattern
 	validModelNamePattern       = validNamespacePattern
 	validTypeSubsectionPattern  = `(` + validResourceTypePattern + `:` + validResourceSubTypePattern + `|remote:)`
-	validResourcePattern        = `^rdk.` + validNamespacePattern + `:` + validTypeSubsectionPattern + `\/` + validModelNamePattern + `$`
+	validResourcePattern        = `^rdk.resource_manager.` + validNamespacePattern + `:` + validTypeSubsectionPattern + `\/` +
+		validModelNamePattern + `$`
```

Comment:
This regex was slightly incorrect (it was correct at an earlier point). "Resource patterns" start with `rdk.resource_manager`, not just `rdk`.

Reply:
Documented all the regexes with examples.